src/utils/AccessibleModal.ts

```diff
@@ -0,0 +1,90 @@
+import { ModalBox as ModalBoxModule } from '../ExternalModulesShim';
+import { FocusTrap } from '../ui/FocusTrap/FocusTrap';
+import { l } from '../strings/Strings';
+import { $$ } from './Dom';
+import { KeyboardUtils, KEYBOARD } from './KeyboardUtils';
+
+export interface IAccessibleModalOptions {
+  overlayClose?: boolean;
+  sizeMod: 'big';
```

Comment:
Does `sizeMod` need to be an option if `big` is the only allowed value?

Reply:
'big' isn't the only option, I inherited this property from `Coveo.ModalBox.Options.sizemod`, which I added on an older PR without knowing what `sizeMod` values are available. I just looked it up and the available values are `'small' | 'normal' | 'big'`, so I'll add that.